### DISCUSSION TITLE
Empty top languages

### DISCUSSION THREAD

**Discussion by elax46** [anuraghazra/github-readme-stats#1790] [Q&A] [answered]
I am also in the same situation as #1640 how can i solve thanks?

![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=elax46&show_icons=true&theme=yeblu)

**Comment by rickstaa** [comment, marked as answer]
@elax46 Similar to #1640 you also don't seem to have any non-forked public repositories that https://github.com/github/linguist recognises. The only repository for which you should get the `js`, `json` and `md` languages is https://github.com/elax46/custom-brand-icons. But Github does not seem to recognise those files (i.e. there is no language information on the repository). You can check this by using the following syntax in the [graphql explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

```graphql
{"login": "elax46"}
```` 

This will give the following response:

```json
{
  "data": {
    "user": {
      "repositories": {
        "nodes": [
          {
            "name": "custom-brand-icons",
            "languages": {
              "edges": []
            }
          },
          {
            "name": "elax46",
            "languages": {
              "edges": []
            }
          }
        ]
      }
    }
  }
}
```

I think your issue is related to https://github.com/github/linguist/issues/542 multiple solutions can be found there.

To find languages in private repositories, you have to deploy your Vercel instance since we do not have access to those repositories (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance for more information).

**Comment by rickstaa** [comment]
Related to #1789.
